fv3core/pace/fv3core/stencils/dyn_core.py

```diff
@@ -823,6 +787,13 @@ def __call__(
                 self.update_geopotential_height_on_c_grid(
                     self._zs, self._ut, self._vt, self._gz, self._ws3, dt2
                 )
+                # TODO (floriand): Due to DaCe VRAM pooling creating a memory
```

Comment:
this is solved, right? The fields are no longer used

Reply:
`self.cgrid_shallow_water_lagrangian_dynamics.ptc` and `self.cgrid_shallow_water_lagrangian_dynamics.delpc` are used directly below. They used to be attached to this object instead of c_sw.